### ISSUE TITLE
Is it possible to edit i18n.js on localehost?

### ISSUE THREAD

**Issue by ahmadta721** [louislam/uptime-kuma#3655]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Hello, I am unfortunately not very familiar, I am a beginner and I really don't know..
I need to edit the i18n.js file on my server.

1- Is it possible to edit i18n.js after installation by Docker?
2- Where is i18n.js file located on my server?

### 🐻 Uptime-Kuma Version

1.23.1

### 💻 Operating System and Arch

Ubuntu 22.04.2

### 🌐 Browser

Version 116.0.5845.111 (Official Build) (64-bit)

### 🐋 Docker Version

Docker version 24.0.1, build 6802122

### 🟩 NodeJS Version

v12.22.9

**Comment by CommanderStorm** [comment]
Yes, you can simply mount a file with the content you desire to the path that you need it to be.
See https://docs.docker.com/storage/bind-mounts/

To find the correct path in the container you can simply exec inside the container `docker exec -it <CONTAINER> <COMMAND>`

Note that doing so is usually not recommended.
If you have a problem that other people also have, bringing code upstream might be a better idea, as otherwise every upgrade will be quite a hastle.
